near-contract-standards/src/non_fungible_token/approval/approval_impl.rs

```diff
@@ -1,10 +1,12 @@
+/// Common implementation of the [approval management standard](https://nomicon.io/Standards/NonFungibleToken/ApprovalManagement.html) for NFTs.
```

Comment:
So small point on these but you only use `///` for when you're documenting the thing below it. You can use `//!` at the top of the file like this to document the module itself